scripts/scil_bundle_diameter.py

```diff
@@ -39,14 +39,20 @@
 from scipy.ndimage import map_coordinates, gaussian_filter
 
 from scilpy.io.streamlines import load_tractogram_with_reference
-from scilpy.io.utils import (add_overwrite_arg, add_reference_arg,
-                             add_json_args, add_verbose_arg,
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_reference_arg,
+                             add_verbose_arg,
+                             assert_headers_compatible,
                              assert_inputs_exist,
                              assert_output_dirs_exist_and_empty,
-                             parser_color_type, snapshot,
-                             assert_headers_compatible)
-from scilpy.viz.scene_utils import create_tube_with_radii
-from scilpy.viz.utils import get_colormap
+                             parser_color_type)
+from scilpy.viz.backends.fury import (create_interactive_window,
+                                      create_scene,
+                                      snapshot_scenes)
+from scilpy.viz.backends.vtk import create_tube_with_radii
+from scilpy.viz.color import get_lookup_table
+from scilpy.viz.screenshot import compose_image
 
 
 def _build_arg_parser():
```

Comment:
FYI, ran its test, with an added --show_rendering: gives
![test_bundle_diameter_show_rendering](https://github.com/scilus/scilpy/assets/4967417/9e6a6579-8459-408d-b34a-9c66fba1579c)

VS on master: seems to be the same image, different view, not centered the same way on page.
![test_bundle_diameter_show_rendering_master](https://github.com/scilus/scilpy/assets/4967417/8db0e7f6-0643-4a63-8208-8ff8b330d0ad)

Reply:
The viewport got flipped and is now placed differently than before, since the script now uses some of the backend functions. The camera gets positioned according to all the remaining viewports in scilpy.

This is a win in my opinion, but I agree the final result is disappointing. I'm not too sure of the bounds that are taken into account here to make it so the screenshot is taken so far from the data. I'll look into that, but I think the usage of `reset_camera` is to blame here. I remember looking into it and from memory the viewport is re-configured using shifty and weird heuristics.